packages/wallet-ui/src/components/pages/Home/Home.view.tsx

```diff
@@ -13,22 +14,25 @@ export const HomeView = ({ address }: Props) => {
   );
   const loader = useAppSelector((state) => state.UI.loader);
   const { upgradeModalVisible } = useAppSelector((state) => state.modals);
-
+  const { getTranslator } = useStarkNetSnap();
+  const translate = getTranslator();
```

Comment:
the pattern should be sth like that
```suggestion
  const { translate } = useMultiLanguages();
```